.vib/percona-xtrabackup/goss/vars.yaml

```diff
@@ -0,0 +1,9 @@
+binaries:
+  - xtrabackup
+root_dir: /opt/bitnami
+sed_in_place:
+  exclude_paths:
+    - /opt/bitnami/mysql/xtrabackup-test/t
```

Comment:
Do you mean `/opt/bitnami/mysql/xtrabackup-test/test` ?

Reply:
The `.../test` path is the one needed for `percona-xtrabackup:8`, but the branch `2.4` uses `.../t`. I could add both paths, but only having `/opt/bitnami/mysql/xtrabackup-test/test` achieves the same.